src/images/block/stars.css

```diff
@@ -0,0 +1,25 @@
+body {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
```suggestion

```

Reply:
body doesn't have default padding, don't write `padding: 0;` for body